web/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/StyleScope.kt

```diff
@@ -153,7 +162,7 @@ open class StyleBuilderImpl : StyleBuilder, StyleHolder {
         } else false
     }
 
-    internal fun copyFrom(sb: StyleBuilderImpl) {
```

Comment:
as for exposing internal functions...
One thing we can do is to mark them `@ComposeWebInternalApi` 
I'm not sure we want to make any stability guarantees about this and similar functions which were internal, so adding an annotation is an option.

Reply:
@mpetuska @eymar I suggest to leave this as exclusively StyleBuilderImpl method and `override val styleScope = StyleScopeBuilder()`

Reply:
That would be ideal, however it's used in the AttrsScope interface scope ([example](http://github.com/mpetuska/compose-jb/blob/28b1dabb848d28fbc7d4c4b52e4e725b4ab57a77/web/core/src/jsMain/kotlin/org/jetbrains/compose/web/elements/Elements.kt#L1019-L1019)) and as such needs to be declared in the interface.